src/pages/CircleAdminPage/CircleAdminPage.tsx

```diff
@@ -370,206 +276,564 @@ export const CircleAdminPage = () => {
-            />
-          }
-        />
-        <ApeToggle
-          {...teamSelection}
-          label="Team Selection Enabled"
-          infoTooltip={
-            <YesNoTooltip
-              yes="Members select a team during allocation and make allocations only to that team"
-              no="Members make allocations to anyone in the circle"
-            />
-          }
-        />
-        <ApeToggle
-          {...autoOptOut}
-          label="Auto Opt Out?"
-          infoTooltip={
-            <YesNoTooltip
-              yes="If a member doesn't make allocations in an epoch, they'll be set to opt out of receiving allocations in the next epoch. They can still opt back in."
-              no="Members' opt-in/opt-out settings will not be changed automatically."
-            />
-          }
-        />
-        {isFeatureEnabled('fixed_payments') && (
-          <FormAutocomplete
-            {...fixedPaymentToken}
-            options={tokens}
-            label="Fixed Payment Token"
-            fullWidth
-          />
-        )}
-      </Box>
-      <AdminIntegrations circleId={circleId} />
-      <div className={classes.bottomContainer}>
-        <p className={classes.subTitle}>Discord Webhook</p>
-        {allowEdit && (
-          <input
-            readOnly={!allowEdit}
-            className={classes.input}
-            {...discordWebhook}
-          />
-        )}
-        <div className={classes.webhookButtonContainer}>
-          {!allowEdit && (
+    <>
+      <Form id="circle_Admin">
+        <SingleColumnLayout>
+          <Flex css={{ justifyContent: 'space-between' }}>
+            <Text h1 css={{ mb: '$md' }}>
+              Circle Admin
+            </Text>
             <Button
-              onClick={editDiscordWebhook}
-              color="neutral"
+              css={{
+                gap: '$xs',
+                alignSelf: 'flex-end',
+              }}
+              color="primary"
               size="medium"
+              type="submit"
+              form="circle_admin"
               outlined
+              disabled={!isDirty}
+              onClick={handleSubmit(onSubmit)}
             >
-              <EditIcon />
-              Edit WebHook
+              Save Settings
             </Button>
-          )}
-        </div>
-      </div>
-      {!isEmpty(errors) && (
-        <Box
-          css={{
-            display: 'flex',
-            flexDirection: 'column',
-            justifyContent: 'center',
-            margin: 0,
-            color: '$alert',
-          }}
-        >
-          {Object.values(errors).map((error, i) => (
-            <div key={i}>{error.message}</div>
-          ))}
-        </Box>
-      )}
-      <Button
-        css={{ mt: '$lg', gap: '$xs' }}
-        color="primary"
-        size="medium"
-        disabled={!isDirty}
-      >
-        <DeprecatedSaveIcon />
-        Save
-      </Button>
-    </Form>
+          </Flex>
+          <Panel
+            css={{
```

Comment:
this css is repeated a lot; please pull it out into a reusable variable or small component